compute/cker/include/cker/train/operation/DepthwiseConv.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __NNFW_CKER_TRAIN_OPERATION_DEPTHWISECONV_H__
+#define __NNFW_CKER_TRAIN_OPERATION_DEPTHWISECONV_H__
+
+#include "cker/eigen/depthwise_conv_op.h"
+#include "cker/Shape.h"
+#include "cker/Types.h"
+
+namespace nnfw
+{
+namespace cker
+{
+namespace train
+{
+
+class DepthwiseConv
+{
+public:
+  DepthwiseConv() = default;
+
+  template <typename T> int64_t kPacketSize() const
```

Comment:
@ragmani 
If `DepthwiseConv` class is a class template, then the `DepthwiseConvolutionLayer` in train backend should have each type of `DepthwiseConv` class. Is this a good approach? Since other things except for this function are almost the same, I only made this function as a template function.

Reply:
I understand why you used function template. You can use polymorphism to make `DepthwiseConvolutionLayer` independent from type of `DepthwiszeConv`, but I think it's not a good approach.

I still think it's better to use class template because there may be ambiguity in using a different type from the kernel functions below when using function template. But it would be better to keep this code as is for now as long as there is no alternative.